documentation/docs/examples/cad-styling.mdx

```diff
@@ -65,6 +65,24 @@ model.changeStyledNodeSetAppearance(myNodeSet, DefaultNodeAppearance.Ghosted);
 model.removeStyledNodeSet(myNodeSet);
 ```
 
+Styling is "additive", meaning that styles from multiple styled sets are accumulated per node, much 
+like how multiple classes are combined when working with CSS. Styled sets are applied first-in first-out (FIFO)
+in the order the are added.
+
+```js
+// import { ByTreeIndexNodeSet, DefaultNodeAppearance, NumericRange } from '@cognite/reveal'
+
+// you can start from binding styles to an empty set and populate it later
+const set1 = new ByTreeIndexNodeSet(new NumericRange(0, 100));
+model.addStyledNodeSet(set1, { color: [ 255, 0, 0 ] });
+const set2 = new ByTreeIndexNodeSet(new NumericRange(50, 100));
+model.addStyledNodeSet(set2, { outlineColor: NodeOutlineColor.White });
+
+// [0..50] will be colored red
+// [50..100] will be colored red and have a white outline
+// [100..150] will have the default color and a white outline
```

Comment:
is there something missing in this example? I don't see 150 anywhere 🤔

Reply:
Actually, `new NumericRange(50, 100)` means [50...150] as the second argument is count, but I agree this is confusing so I'll change to something more clear.